test-harness/scenarios/enabled-rules-amount.oas3.scenario

```diff
@@ -24,7 +24,7 @@ components:
 ====command====
 {bin} lint {document} --ruleset ./rulesets/parameter-description.oas3.yaml -v
 ====stdout====
-Found 48 rules (1 enabled)
+Found 49 rules (1 enabled)
```

Comment:
```suggestion
Found 48 rules (1 enabled)
```

Reply:
@mkistler could you apply the above suggestion?
The amount of rules remained unchanged in the end.

I'd do it on my own, but for some reason I'm experiencing the following error
![image](https://user-images.githubusercontent.com/9273484/106050203-41a55c00-60e7-11eb-89f5-ccfcd45426bf.png)
I suspect I have insufficient permissions.